RuntimeUnityEditor.Bepin5/PatchInspector/PatchInspector.cs

```diff
@@ -0,0 +1,1375 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Text;
+using BepInEx;
+using HarmonyLib;
+using RuntimeUnityEditor.Core;
+using RuntimeUnityEditor.Core.Utils.Abstractions;
+using UnityEngine;
+
+namespace RuntimeUnityEditor.Bepin5.PatchInspector
+{
+	public class PatchInspector : Window<PatchInspector>
+	{
+		private string searchInput = String.Empty;
+		private Vector2 scrollPos;
+		private List<PatchInfo> foundPatches = new List<PatchInfo>();
+		private bool showFilePaths = true;
+
+		private int nextWindowId = 13000;
+		private Dictionary<string, ILViewerWindow> ilViewerWindows = new Dictionary<string, ILViewerWindow>();
+		private Dictionary<string, List<PatchMethodInfo>> opPatchStates = new Dictionary<string, List<PatchMethodInfo>>();
+		protected override void Initialize(InitSettings initSettings)
+		{
+			DisplayName = "Patch Inspector";
```

Comment:
Make it not open at startup
```suggestion
			Enabled = false;
			DisplayName = "Patch Inspector";
```